### PR TITLE
Adding support of storage v2 to Sql auditing

### PR DESCRIPTION
**Pull request by yoavrubin**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@yoavrubin__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yrubin). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj [hunk 1]
```diff
@@ -59,7 +59,7 @@
     </Reference>
     <Reference Include="Microsoft.Azure.Common.Authentication, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.27-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
+      <HintPath>..\..\..\packages\Microsoft.Azure.Common.Authentication.1.0.26-preview\lib\net45\Microsoft.Azure.Common.Authentication.dll</HintPath>
```
Comment (markcowl) [inline src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.csproj]: Please do not downgrade the package

**Comment by markcowl** [comment]
@azuresdkci retest this please

File: src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj [hunk 2]
```diff
@@ -210,6 +210,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.Azure.Common.2.1.0\lib\net45\Microsoft.Azure.Common.NetFramework.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Azure.Management.Storage">
```
Comment (markcowl) [inline src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj]: @yoavrubin You will need to update the installer wix file to install this new dll.  Please rebuild and recreate the installer wix suing tools\BuildInstaller.ps1

File: setup/azurecmdfiles.wxi [hunk 3]
```diff
@@ -44,15 +44,6 @@
                         <Component Id="cmpDC433AC0D6833219D86CA13947A3934D" Guid="*">
                             <File Id="filA95ABA71ABB678931802000F97D03EDA" KeyPath="yes" Source="$(var.sourceDir)\ResourceManager\AzureResourceManager\ApiManagement\Microsoft.Azure.ResourceManager.dll" />
                         </Component>
-                        <Component Id="cmp1A6CF5E9ED315E384B7AB1184EC3175B" Guid="*">
```
Comment (stankovski) [inline setup/azurecmdfiles.wxi]: These files should not have been removed

**Comment by markcowl** [comment]
@yoavrubin The wxi changes delete some files from other RPs - please ensure that you've synced with the dev branch, that you've cleaned and rebuilt, and then regenerate the installer.

**Comment by yoavrubin** [comment]
@markcowl - this is the wxi that was generated exactly after the process you described - sync, clean, rebuild and regenerate

**Comment by markcowl** [comment]
@yoavrubin Can you target this fix at the release branch?

**Pull request closed without merging**